lib/matplotlib/backends/backend_gtk3.py

```diff
@@ -381,6 +393,88 @@ def stop_event_loop(self):
     stop_event_loop.__doc__=FigureCanvasBase.stop_event_loop_default.__doc__
 
 
+_flow = [Gtk.Orientation.HORIZONTAL, Gtk.Orientation.VERTICAL]
+flow_types = ['horizontal', 'vertical']
+
+
+class Window(WindowBase, Gtk.Window):
```

Comment:
I guess "bad" beats "ugly" (ugly in terms of a list of housekeeping aliases at end of each file.  I can't imagine how it could get bad, (on the other hand I found debugging that transform logic a nightmare with multiple layers of transforms to go through and understand what they what they were doing and why they were failing), so meh.